management/src/systemtests/commission_test.go

```diff
@@ -87,23 +87,88 @@ func (s *SystemTestSuite) TestCommissionNodesFailureAlreadyAllocated(c *C) {
 
 func (s *SystemTestSuite) TestCommissionNodeSuccess(c *C) {
 	nodeName := validNodeNames[0]
-	s.commissionNode(c, nodeName, s.tbn1)
+	s.commissionNode(c, nodeName, ansibleMasterGroupName, s.tbn1)
 }
 
 func (s *SystemTestSuite) TestCommissionNodeSerialSuccess(c *C) {
 	nodeName := validNodeNames[0]
-	s.commissionNode(c, nodeName, s.tbn1)
+	s.commissionNode(c, nodeName, ansibleMasterGroupName, s.tbn1)
 	s.checkHostGroup(c, nodeName, "service-master")
 
 	// commission second node and make sure it is added as worker
 	nodeName = validNodeNames[1]
-	s.commissionNode(c, nodeName, s.tbn1)
+	s.commissionNode(c, nodeName, ansibleWorkerGroupName, s.tbn2)
 	s.checkHostGroup(c, nodeName, "service-worker")
 }
 
+func (s *SystemTestSuite) TestCommissionNodeSerialMastersSuccess(c *C) {
+	nodeName := validNodeNames[0]
+	s.commissionNode(c, nodeName, ansibleMasterGroupName, s.tbn1)
+	s.checkHostGroup(c, nodeName, "service-master")
+
+	// commission second node
+	nodeName = validNodeNames[1]
+	s.commissionNode(c, nodeName, ansibleMasterGroupName, s.tbn1)
+	s.checkHostGroup(c, nodeName, "service-master")
+}
+
+func (s *SystemTestSuite) TestCommissionNodesWithoutHostGroupFailure(c *C) {
+	nodeName := validNodeNames[0]
+
+	// commission a worker node directly
```

Comment:
this seems like a copy-paste thing. Is this comment valid? else we shall remove this.